invenio_theme/assets/semantic-ui/js/invenio_theme/truncate.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+$(".truncate-button").on("click", function () {
```

Comment:
I would rather distinguish `.show-more` and `.show-less` buttons, rather than trying to fit all the buttons under one function. Then it will be easier to select the other button to "hide" as in:
```
$(this).siblings('.show-less')
```
I think it would make this event handler more readable in terms of code